content/build/building/multi-platform.md

```diff
@@ -9,36 +9,55 @@ aliases:
 - /mackit/multi-arch/
 ---
 
-Docker images can support multiple platforms, which means that a single image
-may contain variants for different architectures, and sometimes for different
-operating systems, such as Windows.
+A multi-platform image refers to a single image that includes variants for
+multiple different architectures and, in some cases, different operating
+systems, like Windows. This means that whether you are using an ARM-based
+system or an x86 machine, Docker automatically detects and selects the
+appropriate variant for your system's operating system and architecture.
```

Comment:
"system's operating system" looks a bit repetitive? Maybe instead we can use:

```suggestion
appropriate variant for your host's operating system and architecture.
```

Reply:
yeah, thanks!